admin/class-metabox.php

```diff
@@ -702,7 +717,7 @@ function do_meta_box( $meta_field_def, $key = '' ) {
 					$options_count = count( $meta_field_def['options'] );
 
 					// @todo [JRF => whomever] verify height calculation for older WP versions, was 16x, for WP3.8 20x is more appropriate.
-					$content .= '<select multiple="multiple" size="' . esc_attr( $options_count ) . '" style="height: ' . esc_attr( ( $options_count * 20 ) + 4 ) . 'px;" name="' . $esc_form_key . '[]" id="' . $esc_form_key . '" class="yoast' . $class . '">';
+					$content .= '<select multiple="multiple" size="' . esc_attr( $options_count ) . '" style="height: ' . esc_attr( ( $options_count * 20 ) + 4 ) . 'px;" name="' . $esc_form_key . '[]" id="' . $esc_form_key . '" class="yoast' . $class . '"' . $placeholder . '>';
```

Comment:
The esc_attr()` for the style height would be better either as `absint()` or including the `height: ` inside the `esc_attr()` too.